server/src/main/java/org/elasticsearch/script/ScriptContextStats.java

```diff
@@ -18,33 +18,38 @@
 import java.io.IOException;
 import java.util.Objects;
 
-public class ScriptContextStats implements Writeable, ToXContentFragment, Comparable<ScriptContextStats> {
-    private final String context;
-    private final long compilations;
-    private final TimeSeries compilationsHistory;
-    private final long cacheEvictions;
-    private final TimeSeries cacheEvictionsHistory;
-    private final long compilationLimitTriggered;
+public record ScriptContextStats(
```

Comment:
Nit: I understand this was here before you touched it, but if you have a good view of what is this class doing, do you mind adding a javadoc?

Reply:
Thanks!